backend/nlp/handler.py

```diff
@@ -14,11 +32,52 @@ def hello(event, context):
 
     return response
 
-    # Use this code if you don't use the http event with the LAMBDA-PROXY
-    # integration
-    """
-    return {
-        "message": "Go Serverless v1.0! Your function executed successfully!",
-        "event": event
-    }
-    """
+
+
+"""
+"""
+def analyze_text(text, analyzer):
```

Comment:
If this function simply calls polarity_scores, we probably don't need it. Are you expecting this function to get more complex in the future? If not, let's just use polarity_scores directly.

Reply:
Re: I know I mentioned it yesterday but wanted to reply here and now again that yeah we'll be planning on adding on to this function later and making it do the bulke of review manipulation and analyzing